test/ui/InitializationPlaceholderTest.ts

```diff
@@ -85,6 +85,18 @@ export function InitializationPlaceholderTest() {
         expect(oneFacet.hasClass(InitializationPlaceholder.INITIALIZATION_CLASS)).toBe(false);
         expect(oneFacet.hasClass('coveo-with-placeholder')).toBe(false);
       });
+
+      it('should remove placeholder for facet after components are initialized and after the first query error', () => {
```

Comment:
This could be two tests :) If it's splitted when it fails we have a better idea as to why it failed!